docs/user/OIDC/OAuth2OIDC-keycloak.md

```diff
@@ -1,9 +1,310 @@
 # Configuring Keycloak as an OIDC provider
+(created May 17th 2021)
 
-This document explains how to configure Keycloak as an OIDC provider (check general information and pre-requisites for [using an OAuth2/OIDC Provider with Kubeapps](../using-an-OIDC-provider.md)).
+This document explains how to configure Keycloak as an IDP + OIDC provider (check general information and pre-requisites for [using an OAuth2/OIDC Provider with Kubeapps](../using-an-OIDC-provider.md)).
+It covers the installation and documentation for Kubeapps interacting with two AWS clusters. 
 
-In the case of Keycloak, you can find the parameters in the Keycloak admin console:
+The base installation is done using the bitnami chart for Keycloak.
+
+# Installation
+
+## SSL
+
+In order to support OIDC or OAuth, most servers and proxies require HTTPS. By default, the certificate created by the helm chart / keycloak server is both invalid (error with notBefore attribute) and also based on a deprecated certificate version making it incompatible to use (i.e. is it based on Common Name instead of SAN and is rejected).
```

Comment:
```suggestion
In order to support OIDC or OAuth, most servers and proxies require HTTPS. By default, the certificate created by the helm chart / keycloak server is both invalid (error with `notBefore` attribute) and also based on a deprecated certificate version making it incompatible to use (i.e. is it based on Common Name instead of SAN and is rejected).
```